reference/pipeline/stages/index.md

```diff
@@ -13,10 +13,24 @@ sidebar:
 This article describes the currently-supported stages that you can add to your
 Spinnaker pipelines.
 
+Note that when you're creating a pipeline, you probably won't see every stage
+that's listed here: you'll only see the stages that Spinnaker supports on your
+provider.
+
 ## General
 
 ### Bake
-Bake an image in the specified region.
+Bake an image from the specified packages. _Baking_ is the process of
```

Comment:
We also use "bake" to describe creating a manifest with the "Bake (manifest)" stage

Reply:
Hm, understood. What if we changed this to just say "Create a machine image from the specified packages" without defining baking specifically, since it has two meanings? (Or perhaps @dorbin has a better wordsmithing suggestion?) 

I didn't want to talk about the Bake (Manifest) stage here because it felt too much like a tangent, but am open to doing that if you think clarification about the differences would be helpful.

Reply:
Totally agree about it being a tangent -- I was just worried that our definition here prevents us from defining "Bake (Manifest)" later

Reply:
Another option: we could also just say 'Baking here refers to the process of creating a machine image', and then in "Bake (Manifest)" later we can say 'Baking here refers to the process of creating a manifest'. 

What do you think makes the most sense?

Reply:
:+1: to Nora's rewrite&mdash;define bake implicitly by summarizing the stage behavior. There's still a chance of ambiguity between Bake and Bake (Manifest), but I don't think it's _that_ ambigious.

Reply:
:+1: as well